File: PRAK604-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>
#include <string.h>

int main()
{
    char kataA[100], kataB[100];
    scanf(" %[^\n]%*c", &kataA);
    scanf(" %[^\n]%*c", &kataB);
    int lenA, lenB, i, c=0, d=0;
    lenA = strlen(kataA);
    lenB = strlen(kataB);
    if (lenA!=lenB){
        printf("Panjang kalimat berbeda, pesan palsu");
    }
    else{
        for (i=0;i<lenA;i++){
            if(kataA[i]==kataB[i]){
                printf("*");
                c++;
            }
            else{
                printf("#");
                d++;
            }
        }
        printf("\n* = %d", c);
        printf("\n# = %d", d);
        if(c>=d){
            printf("\nPesan Asli");
        }
        else{
            printf("\nPesan Palsu");
        }
    }
}

File: PRAK604-2210817220019-DHEA_APRILINDA_UTAMI.py
kataA = input()
kataB = input()
c=0
d=0
if(len(kataA)!=len(kataB)):
    print("Panjang kalimat berbeda, pesan palsu")
else :
    for i in range (len(kataA)):
        if (kataA[i]==kataB[i]):
            print("*", end="")
            c+=1
        else:
            print("#", end="")
            d+=1
    print()
    print("* =", c)
    print("# =", d)
    if(c>=d):
        print("Pesan Asli")
    else:
        print("Pesan Palsu")